onnx_transformers/pipelines.py

```diff
@@ -555,7 +559,13 @@ def __init__(
             logger.info(f"loading onnx graph from {self.graph_path.as_posix()}")
             self.onnx_model = create_model_for_provider(str(graph_path), "CPUExecutionProvider")
             self.input_names = json.load(open(input_names_path))
-            self.framework = "np"
+            self.framework = "pt"
```

Comment:
This will cause other things to break. Are all tests passing ? 
you can run tests using `make test`.